src/sentry/utils/snuba.py

```diff
@@ -1206,7 +1214,7 @@ def _resolve_column(col: str) -> str:
     return _resolve_column
 
 
-def resolve_condition(cond, column_resolver):
+def resolve_condition(cond: list, column_resolver: Callable[[Any], Any]) -> Sequence[Any]:
```

Comment:
they specify that input can be a tuple, but they do element assignment which would error on tuple, so it can only be a list. I added some asserts for this too